### PR TITLE
feat(logs/page) Add logs page

### LINKED ISSUE #920: Move bulk of chronograf log viewer and required reusable components to platform
Move bulk of chronograf log viewer and required reusable components to platform

### PR DESCRIPTION
**Pull request by OfTheDelmer**
Closes #920  

_Briefly describe your proposed changes:_
Copy over the bulk of log viewer and related components. 

It currently cannot build queries or fetch data. It also does not include logic for org settings management since that isn't part of platform yet.

  - [x] Rebased/mergeable
  - [x] Tests pass

---

- [x] Updated to use `clockface` imports instead of `reusable_ui`
- [x] Component's with stylesheets are in folders that contain their stylesheet
- [x] Component stylesheets are imported in chrongraf scss
- [x] Updated to use UI kit for all buttons, dropdowns, etc so that no instances of old UI components exist in Platform 

---

- [x] Remove or stub components and utils that would not be viewable until querying is working for log viewer: histogram, logs table, and options overlay.

### CODE REVIEW

File: chronograf/ui/src/logs/utils/logFilters.test.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
+import {searchToFilters} from 'src/logs/utils/search'
+import {Operator} from 'src/types/logs'
+
+describe('Logs.searchToFilters', () => {
+  const isUUID = expect.stringMatching(
```
Comment (bthesorceror) [inline chronograf/ui/src/logs/utils/logFilters.test.ts]: Can we move this to a global test helper, seems valuable in many places

File: chronograf/ui/src/logs/utils/matchSections.test.ts [hunk 2]
```diff
@@ -0,0 +1,209 @@
+import {Filter, MatchType} from 'src/types/logs'
+import {getMatchSections, filtersToPattern} from 'src/logs/utils/matchSections'
+
+describe('Logs.matchFilters', () => {
+  const isUUID = expect.stringMatching(
```
Comment (bthesorceror) [inline chronograf/ui/src/logs/utils/matchSections.test.ts]: case and point for previous comment ;)

File: chronograf/ui/src/logs/containers/logs_page/LogsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,1037 @@
...
+    } catch (e) {
+      console.error('Failed to get sources and namespaces for logs')
+    }
+
+    this.updateTableData(SearchStatus.Loading)
+
+    if (getDeep<string>(this.props, 'timeRange.timeOption', '') === 'now') {
+      this.startLogsTailFetchingInterval()
+    }
+    // await this.props.executeHistogramQueryAsync()
```
Comment (bthesorceror) [inline chronograf/ui/src/logs/containers/logs_page/LogsPage.tsx]: why is this commented out? if its not need please just remove it

File: chronograf/ui/src/logs/utils/logFilters.test.ts (hunk 1 above)
Comment (OfTheDelmer) [reply chronograf/ui/src/logs/utils/logFilters.test.ts]: @bthesorceror where are we putting our global test helpers for platform?

**Pull request merged**